service/tbc/tbc.go

```diff
@@ -306,26 +311,36 @@ func (s *Server) seed(pctx context.Context, peersWanted int) ([]tbcd.Peer, error
 	defer cancel()
 
 	errorsSeen := 0
-	var addrs []net.IP
-	for k := range s.seeds {
-		ips, err := resolver.LookupIP(ctx, "ip", s.seeds[k])
+	var moreSeeds []tbcd.Peer
+	for _, v := range s.seeds {
+		host, port, err := net.SplitHostPort(v)
+		if err != nil {
+			log.Errorf("SplitHostPort: %v", err)
```

Comment:
I think we could make this error message a bit easier to debug:
```suggestion
			log.Errorf("Failed to parse host/port: %v", err)
```